vuepress/docs/tutorials/micro-frontends/communication.md

```diff
@@ -0,0 +1,506 @@
+---
+sidebarDepth: 2
+---
+
+::: warning Recommended Learning Path
+1. [Create a React Micro Frontend](react)
+:::
+
+# Communicate Between Micro Frontends
+
+Entando supports communication between micro frontends using [Custom Events](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent), an established web standard.
+
+In this tutorial, we’ll build two micro frontends.
+- The first one will fire an event.
+- The second one will intercept it.
+
+## Publisher
+
+Create a simple app to publish an event.
+
+``` bash
+npx create-react-app publisher-widget --use-npm
+```
+
+Start the app.
+
+``` bash
+cd publisher-widget
+```
+
+``` bash
+npm start
+```
+
+Enter "Y" to run the subscriber-widget on another port.
```

Comment:
1) subscriber-widget? 2) Seems odd to have this block here at all since the errors are self-explanatory.

Reply:
Thanks, I put that in the wrong spot. I thought about it, and I think you're right that the error message is self-explanatory. Deleted.